judgments/utils/search_utils.py

```diff
@@ -0,0 +1,69 @@
+import datetime
+
+from ds_caselaw_utils import courts as all_courts
+
+ALL_COURT_CODES = [court.code for court in all_courts.get_all()]
+
+
+def _valid_years():
+    """
+    Generate a list of valid years as strings.
+    """
+    today = datetime.date.today()
+    valid_years = range(2003, today.year)
+    return [f"{year}" for year in valid_years]
+
+
+def _get_value_as_number(dict_item):
+    """
+    Given a single dictionary item,
+    return the value as an integer.
+    """
+    return int(dict_item[1])
+
+
+def _sort_by_number_in_value(unsorted_dict: dict):
+    """
+    Sorts a dictionary by value where
+    the values contain numbers as strings
+    """
```

Comment:
Does suggest we might just want to make those values ints early in the process.

Reply:
Agree to do this in future, maybe alongside changes to preserve the facet name?

Reply:
yep, that's fine.